deepagent/agent/review_pipeline.py

```
@@ -11,25 +10,27 @@
 os.environ.setdefault("OPENROUTER_API_KEY", config.openrouter_api_key)
 
 from pydantic_ai import Agent  # noqa: E402
+from pydantic_ai.settings import ModelSettings  # noqa: E402
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)